table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -177,31 +182,46 @@ public Table createTable(String tableName, Schema schema) {
    * external synchronization.
    *
    * @param context journal context
-   * @return true if the database changed as a result of fullSync
+   * @param service executor service for parallezing the sync
+   * @return true if the database changed as a result of syncing with UDB
    */
-  public boolean sync(JournalContext context) throws IOException {
-    boolean updated = false;
+  public boolean sync(JournalContext context, ExecutorService service) throws IOException {
```

Comment:
this `ExecutorService` is not used?

Reply:
ooops, I forgot to update `invokeAll`. Thanks!